src/DataGrid.tsx

```diff
@@ -68,58 +54,94 @@ export interface DataGridProps<R, K extends keyof R> {
    * 4. Update all cells under a given cell by double clicking the cell's fill handle.
    */
   onGridRowsUpdated?<E extends GridRowsUpdatedEvent<R>>(event: E): void;
-  /** Called when a column is resized */
-  onColumnResize?(idx: number, width: number): void;
 
-  /** Grid Props */
-  /** The primary key property of each row */
-  rowKey?: K;
+  /**
+   * Dimensions props
+   */
+  /** The width of the grid in pixels */
+  width?: number;
+  /** The height of the grid in pixels */
+  height?: number;
+  /** Minimum column width in pixels */
+  minColumnWidth?: number;
   /** The height of each row in pixels */
   rowHeight?: number;
-  defaultFormatter?: React.ComponentType<FormatterProps<R>>;
-  rowRenderer?: React.ComponentType<RowRendererProps<R>>;
-  rowGroupRenderer?: React.ComponentType;
-  /** A function called for each rendered row that should return a plain key/value pair object */
-  rows: readonly R[];
-  /** The minimum height of the grid in pixels */
-  minHeight?: number;
-  /** Component used to render a context menu. react-data-grid-addons provides a default context menu which may be used*/
-  contextMenu?: React.ReactElement;
-  /** Used to toggle whether cells can be selected or not */
-  enableCellSelect?: boolean;
-  /** Toggles whether cells should be autofocused */
-  enableCellAutoFocus?: boolean;
-  enableCellCopyPaste?: boolean;
-  enableCellDragAndDrop?: boolean;
-  cellNavigationMode?: CellNavigationMode;
-  /** The node where the editor portal should mount. */
-  editorPortalTarget?: Element;
+  /** The height of the header row in pixels */
+  headerRowHeight?: number;
+  /** The height of the header filter row in pixels */
+  headerFiltersHeight?: number;
+
+  /**
+   * Feature props
+   */
+  /** Set of selected row keys */
+  selectedRows?: ReadonlySet<R[K]>;
+  /** Function called whenever row selection is changed */
+  onSelectedRowsChange?(selectedRows: Set<R[K]>): void;
   /** The key of the column which is currently being sorted */
   sortColumn?: keyof R;
   /** The direction to sort the sortColumn*/
   sortDirection?: DEFINE_SORT;
   /** Function called whenever grid is sorted*/
   onGridSort?(columnKey: keyof R, direction: DEFINE_SORT): void;
```

Comment:
Same here, `onSort`?

Reply:
For this one, I would agree with you, we can ignore the `Grid` word like `onSort` only.

Reply:
Nm, for some reason I thought we fixed the line height in a separate PR